source/material_model/rheology/elasticity.cc

```diff
@@ -123,6 +127,11 @@ namespace aspect
           AssertThrow(use_fixed_elastic_time_step == true,
                       ExcMessage("Stress averaging can only be used if 'Use fixed elastic time step' is set to true'"));
 
+        stabilization_scale_factor = prm.get_double ("Stabilization scale factor");
+        if (stabilization_scale_factor < 1)
+          AssertThrow(stabilization_scale_factor < 1,
+                      ExcMessage("Stabilization scale factor must be greater than one"));
```

Comment:
You can simplify this in the following way:
```suggestion
          AssertThrow(stabilization_scale_factor >= 1.,
                      ExcMessage("Stabilization time scale factor must be greater than or equal to one."));
```

Technically this will never happen, because you declared the minimum allowed value in the `declare_parameters` function already (you can check that by trying to set it lower and commenting out this line). However it does not hurt to keep it to make this point extra obvious.